packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -285,8 +292,14 @@ const withChunking = (config) => {
                         // 2. The package is one of the monorepo packages.
                         //    This is for local development to ensure the bundle
                         //    composition is the same as a production build
+                        // 3. If extending another template, don't include the
+                        //    baseline route files in vendor.js
                         test: (module) => {
-                            if (module?.context?.match(makeRegExp(`${sysPath}/node_modules`))) {
+                            if (
+                                EXT_EXTENDS &&
+                                EXT_OVERRIDES_DIR &&
+                                module?.context?.includes(`/${EXT_EXTENDS}/`)
```

Comment:
It looks like `EXT_EXTENDS` is an array, do you mean `EXT_OVERRIDES_DIR`? also, why do you need the leading and trailing slash?

Reply:
slashes are just to reduce risk of name collision since we know that's the exact npm dependency name. 

`EXT_EXTENDS` is not an array, we coerce it to one in other places